frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx

```diff
@@ -9,14 +9,17 @@ import TooltipIcon from 'components/common/TooltipIcon';
 interface IAppraisalCheckListFormProps {
   className?: string;
   isReadOnly?: boolean;
+  taskStatusCode?: string;
```

Comment:
since there are multiple copies of the Appraisal tasks, this allows the parent component to specify which one we want (the optional or required variant).